scenarios/kubernetes_e2e.py

```diff
@@ -724,9 +757,13 @@ def create_parser():
         '--kops-state', default='s3://k8s-kops-jenkins/',
         help='Name of the aws state storage')
     parser.add_argument(
+        '--kops-state-gce', default='gs://k8s-kops-gce-jenkins/',
+        help='Name of the kops state storage for GCE')
+    parser.add_argument(
         '--kops-zones', help='Comma-separated list of zones else random choice')
     parser.add_argument(
         '--kops-build', action='store_true', help='If we need to build kops locally')
+    parser.add_argument('--kops-version', help='URL containing kops version to test')
```

Comment:
seems scenario did not use this var - you can drop the flag and it will pass through to kubetest